m1-runner/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# chia-runner/tasks/main.yml
+
+- name: Insert the proper path for gnu-tar in .zshenv.
+  lineinfile:
+    path: "~/.zshenv"
+    line: export PATH="/opt/homebrew/opt/gnu-tar/libexec/gnubin:$PATH"
+    insertbefore: EOF
+
+- name: Source the ~/.zshenv file
```

Comment:
Have you tried without this? Once we identified the correect file, this should just be sourced automatically

Reply:
Sourced automatically after a line has been appended to it?

Reply:
Oh hmm. Thats a good question - not sure if its loading that for every command that gets run or not - may not actually be, though if you end up testing, would be curious to know the result, but this is fine as is then